sklearn/cluster/tests/test_optics.py

```diff
@@ -208,231 +208,87 @@ def test_cluster_sigmin_pruning(reach, n_child, members):
+       0.02428171, 0.01232994, 0.04752247, 0.06648731, 0.01923359,
+       0.06478506, 0.02050194, 0.02570519, 0.00626935, 0.04566003,
+       0.07229984, 0.05972662, 0.12402862, 0.05786566, 0.12902458,
+       0.19397117, 0.06481248, 0.19744853, 0.13050219, 0.07368615,
+       0.10293114, 0.02518936, 0.04737045, 0.07272921, 0.10337971,
+       0.05448018, 0.04560843, 0.37185355, 0.17882154, 0.06383366,
+       0.09666484, 0.15841938, 0.02802419, 0.07680612, 0.05505145,
+       0.04726204, 0.04298729, 0.14588457, 0.05204779, 0.08758364,
+       0.03986742, 0.0404145 , 0.02376484, 0.04621751, 0.15894636,
+       0.06739961, 0.14745807, 0.07519989, 0.10413549, 0.03118367,
+       0.27884321, 0.12577852, 0.01946909, 0.07165066, 0.03484924,
+       0.07072021, 0.06056985, 0.0289722 , 0.04860705, 0.27330833,
+       0.19253633, 0.07473675, 0.09600461, 0.03133039, 0.06652879,
+       0.06213692, 0.12765036, 0.06305507, 0.03230761, 0.07761318,
+       0.10812441, 0.12604482, 3.38166132, 0.07954152, 0.06355475,
+       0.0718342 , 0.09100107, 0.06351387, 0.03578337, 0.09761135,
+       0.08706804, 0.1112927 , 0.15610375, 0.03129849, 0.14103905,
+       0.05477851, 0.08592214, 0.0985648 , 0.38047783, 0.07056325,
+       0.08912907, 0.07893207, 0.07048013, 0.12146806, 0.18014014,
+       0.07106212, 0.04583092, 0.05484359, 0.13149053, 0.08295075,
+       0.05190986, 0.04277681, 0.09795172, 0.05191173, 0.06240285,
+       0.08663622, 0.0387884 , 0.0314522 , 0.08132232, 0.10009446,
+       0.40594025, 0.6958039 , 1.02272736, 1.01009808, 0.36132809,
+       1.40000558, 0.50876207, 0.35528016, 0.41655233, 1.54185878,
+       0.58283302, 1.08161647, 0.79083769, 0.21787959, 0.42869318,
+       0.43233582, 0.77078842, 0.42731442, 0.43857058, 0.43947967,
+       0.44450538, 0.21250175, 1.3021191 , 0.07902971, 0.29901135,
+       0.43341179, 0.48832866, 1.50014159, 0.40381285, 0.71428016,
+       1.67998055, 0.56486474, 0.29504376, 0.99582401, 2.73126254,
+       0.63858302, 0.49901512, 0.1625456 , 0.29009947, 0.41233532,
+       0.28210301, 0.45252974, 0.98694788, 0.84519324, 0.7537702 ,
+       0.4332758 , 0.31252018, 1.44771384, 1.69060132, 0.36163728,
+       2.42023675, 0.71811649, 1.46964434, 0.27110825, 0.58160667,
+       0.30436792, 0.29073531, 0.89860137, 0.36691359, 0.93946811,
+       1.31842005, 0.68243415, 0.1456141 , 1.0761098 , 0.41250304,
+       0.44101933, 0.88379282, 0.29976315, 0.48026722, 0.44065701,
+       0.3953454 , 0.20985811, 0.11029291, 1.10077171, 0.85881447,
+       0.35109906, 0.59462279, 0.79243895, 0.97558335, 1.46257997,
+       0.54302544, 0.7990609 , 0.73341026, 0.54179151, 1.22279402,
+       0.60442441, 0.99768341, 0.59304996, 0.21894289, 0.62705249,
+       0.39311448, 0.70272104, 0.81815493, 0.51196661, 0.63019314,
+       0.62327168, 1.09937516, 0.64112772, 0.6290978 , 0.51004568]
+    
+    # we compare to truncated decimals, so use atol
+    assert_allclose(clust.reachability_, np.array(v), atol=1e-5)
```

Comment:
any reason to increase the tolerance from the default?

Reply:
The values are copy and pasted rather then actually calculate in the test. We can't calculate the other code directly in the test because the python chemometria port has a dependency on hcluster which isn't importable in the test environment. Originally, I wrote out the chemometria values to a numpy array and included them in the test folder, but wasn't able to read them when the tests where run after pushing them to github; hence, why the values are just copy pasted. Python does it's own rounding when printing out numbers to screen, and sometimes will print a value like '0.4246995' (7 digits to the right of the decimal) instead of '0.64324688' (8 digits to the right of the decimal). I assume that whatever the last digit is has been rounded, hence the specified tolerance... that said, it may be possible to change it to 1e-6. The default is atol=1e-08, which won't work without formatting the test array output from the chemometria port differently.